src/components/forms/DebouncedCombobox.tsx

```diff
@@ -161,6 +161,8 @@ export function DebouncedCombobox(props: Props) {
 
       if (props.clearInputAfterSelection) {
         setSelectedOption({ record: internalRecord, withoutEvents: true });
+
+        debouncedSearch('');
```

Comment:
The reason for adding this line of code is the bug we had when selecting values ​​when `clearInputAfterSelection` is true. When we selected one of them, it filtered list using value from selected item.

Reply:
Can you demo this using video or something? I am not sure we want to "clear after input selection"?